path_test.go

```diff
@@ -657,3 +657,27 @@ store:
 	// OUTPUT:
 	// [john ken]
 }
+
+func AnchorPath_PathString() {
+	yml := `
+store:
+  book:
+    - author: &john
+		name: john
```

Comment:
Please indent should be two spaces
```
book:
  - author: &john
      name: john
      best_seller: true
    price: 10
```

Reply:
Oops, this was a tab instead of a space, should be fixed now.